<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Search Properties</string>
    </property>
    <widget class="QCheckBox" name="flagsCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Flags</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="srcCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>src Factor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dstCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>dst Factor</string>
     </property>
    </widget>
    <widget class="QWidget" name="dstContainer" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>235</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>dstFactor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <item>
          <property name="text">
           <string>==</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="srcContainer" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>234</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>srcFactor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>==</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="flagsContainer" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>366</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="flagsHorizontalLayout">
       <item>
        <widget class="QLabel" name="flagsLabel">
         <property name="text">
          <string>Flags &amp; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="flags1LineEdit"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>==</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="flags2LineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchPushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="directoryLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearPushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Directory</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMaterial_XML">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Folder"/>
   </widget>
   <addaction name="menuMaterial_XML"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
